CHANGES.rst

```diff
@@ -1,3 +1,145 @@
+
+4.0
+---
+
+Incorporate changes from workalendar v5.2.2. (2019-07-07)
+
+- **Deprecation Warning:** *Currently the registry returns `OrderedDict` objects when you're querying for regions or subregions. Expect that the next major release will preferrably return plain'ol' `dict` objects. If your scripts rely on the order of the objects returned, you'll have to sort them yourself.*
```

Comment:
It looks like this warning appears twice. Maybe one copy can be eliminated?

Reply:
Good catch.